encodings/dict/Cargo.toml

```diff
@@ -30,6 +30,7 @@ workspace = true
 [dev-dependencies]
 criterion = { workspace = true }
 rand = { workspace = true }
+vortex-array = { workspace = true, features = ["test_harness"] }
```

Comment:
I think features are typically kebab case? Might be wrong, can change this later though, don't block the PR

Reply:
Done.

(Seemed slightly shitty that test-harness gave you a module named test_harness 🤷. @gatesn Should I fix `canonical_counter` in a separate PR?)